content/blog/2016/2016-01-10-beautiful-jenkins-dashboard.adoc

```diff
@@ -0,0 +1,20 @@
+---
+layout: post
+title: "A beautiful Jenkins dashboard"
+author: kj187
+---
+
+
+image:http://res.cloudinary.com/kj187/image/upload/c_scale,w_890/v1450212394/kj187_Dashboard_2_i4wkw6.png[Dashboard, 250, float="right", link="goo.gl/X3WM3r"]
```

Comment:
The image should just be checked in; why bother using a short-link instead of the full link?